README.md

```diff
@@ -33,9 +33,11 @@ To run unit tests execute the following command:
 * The *JENKINS_ARTIFACTORY_URL* environment variable should be set to the Artifactory URL.
 * The *JENKINS_ARTIFACTORY_USERNAME* environment variable should be set to the Artifactory username.
 * The *JENKINS_ARTIFACTORY_PASSWORD* environment variable should be set to the Artifactory password.
+* The *JENKINS_ARTIFACTORY_DOCKER_DOMAIN* environment variable should be set to the Artifactory Docker domain.
+* The *JENKINS_ARTIFACTORY_DOCKER_REPO* environment variable should be set to the Artifactory Docker repository.
```

Comment:
Why is this environment variable necessary? Why not using a hard-coded repository instead?

Reply:
Check the CLI for the same information. You can't hard coded it.